app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -64,6 +64,26 @@ class DradisDatatable {
       },
       lengthChange: false,
       pageLength: 25,
+      stateSave: true,
+      stateSaveCallback: function(settings, data) {
```

Comment:
Let's defined this function as a class function to keep the dataTable definition simple